Gemfile

```diff
@@ -2,5 +2,7 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 gemspec
 
+gem 'aruba'
```

Comment:
Why is this needed, looks redundant:
https://github.com/voxpupuli/modulesync/blob/7202e6bfc819a8e7323a03c9391cf45b2b6df9d2/modulesync.gemspec#L20